content/en/docs/collector/configuration.md

```diff
@@ -502,13 +502,17 @@ consists of three sub-sections:
 - pipelines
 - telemetry
 
+### Extensions
```

Comment:
```suggestion
### Extensions  {#service-extensions}
```

This _should_ fix the markdown issue: by doing that you set a custom anchor, this will not show up in the TOC or heading itself